CMakeLists.txt

```diff
@@ -124,6 +125,14 @@ ELSEIF(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     ELSE()
         MESSAGE(WARNING "\nCannot determine the version of the compiler selected to build GNU Radio (${APPLE_STR}Clang : ${CMAKE_CXX_COMPILER}). This build may or not work. We highly recommend using Apple Clang version ${APPLECLANG_MIN_VERSION} or more recent, or Clang version ${CLANG_MIN_VERSION} or more recent.")
     ENDIF()
+    # Same flags as gcc. Note that -Og has a different meaning for Clang, but
```

Comment:
Ignoring this issue I do not think there is any reason to differentiate beween GCC and clang.

Reply:
The reason for having this twice is that we have separate branches for gcc and clang flags already. MSVC has different flags, so these won't work. I preferred doing this over having a separate "if gcc or clang" branch, but I could also go that route.

Reply:
Clang is fairly compatible with GCC, but the reverse isn't generally true. Clang dev intentionally try to keep compatibility with GCC, while also adding in their own tweaks and options. This, while in general we do use the same settings for both Clang and GCC, we could decide to use different ones some day.